browser/src/plugin.ts

```diff
@@ -82,6 +82,9 @@ export default class ExtismPlugin {
       env: environment
     };
     this.module = await WebAssembly.instantiate(this.moduleData, env);
+    if (this.module.instance.exports._start) {
+      wasi.start(this.module.instance);
```

Comment:
I don't think we want to call `_start` here, since that will actually run the plugin, it should be called in `Plugin.call` instead

Reply:
@zshipko so, you mean it should adopt the same methodology we use in the Rust runtime?